projects/search_extensions.md

```diff
@@ -0,0 +1,42 @@
+# Search dependencies and create custom operators with Sourcegraph Extensions
+
+Sourcegraph's search is getting a lot more powerful and extensible. In addition to [code, commit, and diff search](https://docs.sourcegraph.com/user/search), you can now:
+
+- Search for all JavaScript/TypeScript files that `import` or `require` a package with <code>js.depends:<em>mypackage</em></code>
+- Search for all Go files that import a package with <code>go.imports:<em>mypackage</em></code>
+- ...and do the same for Python, PHP, and Java.
+
+In what cases would you want to search for files that import a particular package?
+
+- You want to remove a dependency and want to see all places it's actually called.
+- Look for examples in your codebase on how to use a package.
+- Deciding which package to use based on how commonly used it is in your codebase.
```

Comment:
These bullet points are not parallel:

- 1st is a sentence
- 2nd is a command
- 3rd is a noun phrase

You should make them all the same kind of phrase